resolver/resolver.go

```diff
@@ -151,7 +151,20 @@ func (a Address) Equal(o Address) bool {
 
 // String returns JSON formatted string representation of the address.
 func (a Address) String() string {
-	return pretty.ToJSON(a)
+	var sb strings.Builder
+	sb.WriteString(fmt.Sprintf("{Addr: %v, ", a.Addr))
+	sb.WriteString(fmt.Sprintf("ServerName: %v, ", a.ServerName))
```

Comment:
Nit: Please see if you can use `%q`. It helps to get to the values of interest when you are looking at a long log line.